srcpkgs/nerd-fonts/template

```diff
@@ -1,14 +1,14 @@
 # Template file for 'nerd-fonts'
 pkgname=nerd-fonts
-version=2.2.1
+version=2.2.2
 revision=1
 depends="font-util xbps-triggers nerd-fonts-ttf nerd-fonts-otf"
```

Comment:
`font-util xbps-triggers` is transitive over `nerd-fonts-ttf nerd-fonts-otf`

Reply:
I'm a bit confused, do you mean that I don't need to include `font-util xbps-triggers` because the other two implicitly provide it?